charts/renku/values.yaml

```diff
@@ -588,6 +588,23 @@ tests:
   image:
     repository: renku/tests
     tag: '0.2.0-rc4'
+  users_json: |
+    [
+      {
+        "username": "bwayne",
+        "firstName": "Bruce",
+        "lastName": "Wayne",
+        "email": "bwayne@example.com",
+        "password": "IamBatman"
```

Comment:
I suppose if these go in the `values.yaml` we should either a) make sure that the users are deleted once the test is done or b) provide a different `users.json` for deployments.

Reply:
I think we may want to tell folks to be careful if they want to run `helm test` in production

Reply:
or template the password? We can leave it out of the default `values.yaml` and use the `required` function when evaluating in https://github.com/SwissDataScienceCenter/renku/pull/412/files#diff-3f630e831a60063e0ab2d4a02e70c2acR28